packages/ember-glimmer/tests/integration/binding_integration_test.js

```diff
@@ -0,0 +1,89 @@
+import { RenderingTest, moduleFor } from '../utils/test-case';
+import { Component } from '../utils/helpers';
+import { set } from 'ember-metal/property_set';
+import { Binding } from 'ember-metal/binding';
+
+moduleFor('Binding integration tests', class extends RenderingTest {
+
+  ['@test bound values should update after the parent element is removed and then re-appended']() {
```

Comment:
I talked to @rwjblue about this, the thing this was testing was not supported anymore (passing a view instance to the `{{view}}` helper). So we can just 🔥 this whole test 😄